docs/hpc/03_storage/10_sharing_data_on_hpc.md

```diff
@@ -63,8 +66,10 @@ $ setfacl -k <file/dir>
 $ setfacl -b <file/dir>
 ```
 
-### Important: Give Access to Parent Directories in the Path
+:::tip
+Give Access to Parent Directories in the Path
 When you would like to set ACL to say `/a/b/c/example.out`,  you also need to set appropriate ACLs to all the parent directories in the path. If you want to give read/write/execute permissions for the file `/a/b/c/example.out`, you would also need to give at least r-x permissions to the directories: `/a`,  `/a/b`, and `/a/b/c`.
+:::
 
 ### Remove All ACL Entries
 ```sh
```

Comment:
The next line could use `$` instead of `#`.

Reply:
Good catch!  Thanks, I totally missed that.
I found a few instances of this error.  Please let me know if I've missed any.